plugins/powerd/meson.build

```diff
@@ -0,0 +1,26 @@
+if host_machine.system() == 'linux'
+  cargs = ['-DG_LOG_DOMAIN="FuPluginPowerd"']
+
+shared_module('fu_plugin_powerd',
```

Comment:
do we want to build this on all systems, e.g. Fedora, Ubuntu, Arch etc? isn't powerd really ChromeOS-specific? If that's true, maybe we need an option in meson_options.txt which gets enabled in ChromeOS (and possibly one of the CI targets for testing). If it's just a D-Bus service which fails on startup then I guess it's fine to leave this compiled on all distros if it makes it easier to test things.

Reply:
I was thinking about this, and I think ideally what we want is a meson option like `plugin_power` that can be set to `none`, `upower`, `powerd`.  There is never going to be a situation that you want both plugins running in Fedora for example.  It's going to be you use one or the other or none.
For the purpose of our CI we can run it in `powerd` mode for something like the Arch CI, in `none` mode for the purpose of Windows, and then `upower` for all the rest.

Reply:
on the fiip side, if one of the plugins self-disable at startup it's one less configure option and one step closer to "just works". I also wonder if arch (or another CI target) can actually run `powerd` for testing.

Reply:
That's true.  I guess let'd hold this decision until there is more meat to the plugin.

Reply:
Hi @superm1 and @hughsie 

This plugin will be similar to the upower plugin in that it will gracefully disable itself on startup if the dbus interface for powerd that we're building isn't there.

So on Chrome OS, the upower plugin comes up as Disabled because it fails to connet to the Upower daemon over debus, and the same will be true on other distros for the powerd plugin.